test/cases/resolution.css

```diff
@@ -5,7 +5,7 @@
 
 @media (min-resolution: 144dpi) { }
 
-@media (min-resolution: 2x) { }
+@media (min-resolution: 2dppx), (min-resolution: 2x) { }
```

Comment:
Can you also add `@media (min-resolution: 2x) {}` use case?

Reply:
I tried to add it, but it fails in the `removes unnecessary prefixes` test.

I don't know how to handle it better?

Reply:
Can you add test, so I will see an error on CI?

Reply:
Yeah, https://github.com/postcss/autoprefixer/runs/7377361678?check_suite_focus=true

Reply:
Hm. Do we need to add `2dppx`? Seems like all browsers which need `2dppx` also works with prefixed values.

Reply:
Do you mean delete the x units and keep only the dppx?

Reply:
Input:

```css
@media (min-resolution: 2x) { }
```

My suggested output:

```css
@media (-webkit-min-device-pixel-ratio: 2), (-o-min-device-pixel-ratio: 2/1), (min-resolution: 2x) { }
```

Reply:
If the browser is configured with `chrome >= 29`, then only the following will be output:

```
@media (min-resolution: 1x) { }
```

But this syntax is only supported by `Chrome >= 68`.

See test case: https://codepen.io/yisi/pen/rNdjdVp

Reply:
So output `dppx` units at the same time as this PR to prevent browsers from not supporting it.

Reply:
Seems like these problems are coming because we are mixing prefixes and polyfills here. As a prefixer, Autoprefixer should remove all code which it added.

* What do you think of adding `2x` support to `postcss-present-env` instead?
* Another option is to add `2x` support to Autoprefixer but do not insert `2dppx`. Instead, we can ask users to add polyfill manually (or print a warning after checking `result.processor.plugins`).

Reply:
Well, I think it's okay to print a warning.

Reply:
We can use this logic:

If:
1. There is `2x` syntax
2. And no `2dppi`
3. And there is specific Chrome versions (we can hardcore versions until Can I Use will merge PR)

We will ask to add `postcss-preset-env` and put it before Autoprefixer

But we may need to create an issue or a plugin for `postcss-preset-env` (you can create this plugin as you did for minmax and ask them to add it to preset).

Reply:
This would be easy to implement with our new media query parser : https://www.npmjs.com/package/@csstools/media-query-list-parser

Happy to create a plugin for this if work hasn't started yet @yisibl?